docs/Design-Entscheidungen/design-entscheidungen.md

```diff
@@ -165,3 +165,38 @@ Die Verwendung von PlantUML bietet mehrere Vorteile für das Projekt, einschlie
 - Open Source: PlantUML ist ein Open-Source-Tool und bietet daher eine breite Community-Unterstützung und regelmäßige Updates.
 
 - Export: PlantUML bietet eine Vielzahl von Exportoptionen, um die erstellten Diagramme in verschiedenen Formaten zu exportieren, z.B. als Bild-, PDF- oder ASCII-Art-Datei.
+
+## Anzeigename-Richtlinien
+
+Für Anzeigenamen gelten die folgenden Richtlinien:
+
+- Ein Anzeigename kann zwischen 5 und 20 Zeichen enthalten
```

Comment:
```suggestion
- Ein Anzeigename hat eine Länge zwischen 5 und 20 Zeichen
```
@E3FxGaming kann verstehe ich immer als "muss man nicht", in unserem Fall muss man aber zwischen 5 und 20